jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -69,11 +83,13 @@ def process(self, input_table):
             wfs.input_2.close()
 
             # Update necessary meta info in the output
-            output_model.meta.cal_step.wfs_combine = 'COMPLETE'
-            output_model.meta.asn.pool_name = asn_table['asn_pool']
-            output_model.meta.asn.table_name = os.path.basename(input_table)
+            output_model.meta.cal_step.wfs_combine = "COMPLETE"
+            output_model.meta.asn.pool_name = asn_table["asn_pool"]
+            output_model.meta.asn.table_name = os.path.basename(input_table)  # noqa: PTH119
```

Comment:
I know I said on the other PR to avoid making pathlib changes, but this one looks small and simple enough that I think it's ok to put it in there.  Plus the test coverage is probably better than in the scripts - I think our regtests should catch if there are any differences in the output metadata.